set.mm

```diff
@@ -419171,116 +419164,181 @@ the theorem forms ("i", "d" or "g") should be the first of multiple
        <td> ` (/) ` </td><td>Yes</td>
        <td> ~ n0i , ~ vn0 ; ~ snnz , ~ prnz </td></tr>
        </table>
-
+       </p>
        </HTML>
 
        (Contributed by DAW, 27-Dec-2016.)  (New usage is discouraged.) $)
-    conventions-label $p |- ph $=
+    conventions-labels $p |- ph $=
       (  ) B $.
   $}
 
-
   ${
-    conventions-contrib.1 $e |- ph $.
+    conventions-comments.1 $e |- ph $.
     $(
        <HTML>
        <p>
-       These conventions are more specifically directed towards contributors.
-       For general conventions, see ~ conventions , and for conventions
-       related to labels, see ~ conventions-label .
+       The following gives conventions used in the Metamath Proof Explorer
+       (MPE, set.mm) regarding comments, and more generally nonmathematical
+       conventions.
+       For other conventions, see ~ conventions and links therein.
        </p>
 
        <ul>
-       <li><b>Contributors.</b>
-       The (original) contributor of an assertion (theorem, definition or
-       axiom) is the first one who proved/provided the assertion and added it
-       to set.mm. She/he is mentioned in the "Contributed by" parenthetical
-       within the comment of the theorem, see Metamath Book, p. 142.
-       An exception should be made if a theorem is an (essential) extract or a
-       variant of an already existing theorem.  An example of the latter is
-       the provision of a theorem in deduction form as variant of a theorem in
-       inference form, e.g., ~ mapsn and ~ mapsnd .  In these cases, the
-       contributor of the former theorem (whose proof can often be shortened by
-       the new theorem) should also be the contributor of the new theorem.  The
-       one who provided the new theorem should be mentioned in the "Revised by"
-       parenthetical (see Metamath Book, p. 143) within the comment of the new
-       theorem.
+       <li><b>Input format.</b>
+       <p>
+       The input format is ASCII.  Tab characters are not allowed.  If
+       non-ASCII characters have to be displayed in comments, use embedded
+       mathematical symbols when they have been defined (e.g.,
+       <pre>` -> `</pre> for ` -> ` ) or HTML entities (e.g., "&amp;eacute;"
+       for "&eacute;").  Default indentation is by two spaces.  Lines are
+       hard-wrapped to be at most 79-character long (this can be achieved,
```

Comment:
80 characters, if the new line counts as well.  See https://stackoverflow.com/questions/32189491/is-the-new-line-character-part-of-the-same-line-as-the-previous-character

Reply:
I had left the "79" that was there, but indeed it is 80, so changed it in (all?) 5 places.